ui/public/mobileapp.html

```diff
@@ -0,0 +1,76 @@
+        <div class="container-fluid">
+          <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
+          <alerts alerts="alerts"></alerts>
+          <div ng-if="!loaded" class="mar-top-xl">Loading...</div>
+          <div ng-if="mobileapp">
+            <h1 class="contains-actions">
+              <div class="pull-right dropdown">
+                <button type="button" class="dropdown-toggle btn btn-default actions-dropdown-btn hidden-xs" data-toggle="dropdown">
+                  Actions
+                  <span class="caret"></span>
+                </button>
+                <a href=""
+                   class="dropdown-toggle actions-dropdown-kebab visible-xs-inline"
+                   data-toggle="dropdown"><i class="fa fa-ellipsis-v"></i><span class="sr-only">Actions</span></a>
+                <ul class="dropdown-menu dropdown-menu-right actions action-button">
+                  <li>
+                    <a ng-href="project/{{project.metadata.name}}/create-mobileapp"
+                       role="button">Create Mobile App</a>
+                  </li>
+                  <li>
+                    <a ng-href="{{mobileapp | editYamlURL}}" role="button">Edit YAML</a>
+                  </li>
+                  <li>
+                    <delete-link
+                      kind="Mobileapp"
+                      resource-name="{{mobileapp.metadata.name}}"
+                      project-name="{{mobileapp.metadata.namespace}}"
+                      alerts="alerts">
+                    </delete-link>
+                  </li>
+                </ul>
+              </div>
+              <span class="ng-class:[mobileapp.metadata.annotations['mobile.k8s.io/iconClass']]"></span> {{mobileapp.metadata.name}}
+              <small class="meta">created <span am-time-ago="mobileapp.metadata.creationTimestamp"></span></small>
+            </h1>
+            <labels labels="mobileapp.metadata.labels" clickable="true" kind="mobileapps" project-name="{{mobileapp.metadata.namespace}}" limit="3"></labels>
+          </div>
+        </div>
+        <div class="container-fluid">
+          <div class="row" ng-if="mobileapp">
+            <div class="col-md-12">
+              <uib-tabset>
+                <uib-tab active="selectedTab.integrate">
+                  <uib-tab-heading>Sync SDK Integration</uib-tab-heading>
+                  <div class="resource-integrate">
+                    <h3>Getting Started</h3>
+                    <p>To start with an example Sync app, clone/fork the repo at <a href="https://github.com/feedhenry-templates/feedhenry-cordova-sync-app">https://github.com/feedhenry-templates/feedhenry-cordova-sync-app</a>. If you already have an App, you can still follow the below instructions to integrate the Sync SDK.</p>
+                    <h3>SDK Config</h3>
+                    <p>Copy the below contents in sync-config.json in the root of the project.</p>
+                    <pre>{{syncconfig}}</pre>
```

Comment:
Yes, the binding secret for sync needs to be created.
I think we can add a message here if the binding secret is not there yet.